components/Trade/Advanced/index.tsx

```diff
@@ -10,16 +10,18 @@ const TradingPanel = styled.div`
     width: 25%;
     display: flex;
     flex-direction: column;
-    height: 92vh;
```

Comment:
does changing the height of this cause everything to squash up??

Reply:
the idea is to reduce the height so it doesn't take up the entire height as in the designs

Reply:
ahhhh yeah, just seems to be lots of wasted space. There should be a way to make it to be the full height to allow the children items to fill the space if they need to

Reply:
the footer*